server/src/main/java/org/opensearch/extensions/BooleanResponse.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.extensions;
+
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.transport.TransportResponse;
+import java.io.IOException;
+import java.util.Objects;
+
+/**
+ * Generic boolean response indicating the status of some previous request sent to the SDK
+ *
+ * @opensearch.internal
+ */
+public class BooleanResponse extends TransportResponse {
```

Comment:
We can use the same response for IndicesModuleNameresponse and get rid of that file?

Reply:
Also, a better name here `ExtensionResponse`?